clients/web/src/views/App.js

```diff
@@ -221,7 +223,8 @@ var App = View.extend({
 
             settings = _.extend(settings, {
                 el: this.$('#g-app-body-container'),
-                parentView: this
+                parentView: this,
+                brandName: this.brandName
```

Comment:
I like the idea of a settings registry because then we can flexibly add things to it as needed in future PRs without affecting every body view, etc.